examples/tutorials/advance_model_config/01_microbatch.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<p><strong>can fit memory constraints -&gt; can fit into memory</strong></p><p>If <code>microbatch</code> on, -&gt; If <code>microbatch</code> is on,</p><p> </p><p>(more operations in model: <em>add_grads</em>, <em>zero_grad</em>). : add some verbs and conjunctions </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>